docs/monitoring.md

```diff
@@ -0,0 +1,221 @@
+
+Counter � counts the number of specific events 
+
+Gauge � shows current state of the ovms or model 
+
+Histograms � shows distributions of counters per range 
+
+Summary � shows quantiles � will not be used on the server side 
+
+For more information please visit [Prometheus metrics detailed description](https://prometheus.io/docs/concepts/metric_types/)
+
+## Enable metrics
+
+By default, the metrics feature is disabled during standard server usage.
+
+To enable default metrics set you need to specify the metrics_enabled flag or json setting:
+
+CLI
+
+   ```bash
+         docker run --rm -d -v ${PWD}/models/resnet-50-tf:/opt/model -p 9001:9001 -p 9002:9002 openvino/model_server:latest \
+               --model_path /opt/model --model_name resnet --port 9001 \
+               --rest_port 9002 \
+               --metrics_enabled
+   ```
+
+CONFIG CMD
+
+   ```bash
+         docker run --rm -d -v -d -v ${PWD}/workspace:/workspace openvino/model_server --config_path /workspace/config.json -p 9001:9001 -p 9002:9002 openvino/model_server:latest \
+               --rest_port 9002
+   ```
+
+CONFIG JSON
+
+   ```bash
+   {
+    "model_config_list": [
+        {
+           "config": {
+                "name": "resnet",
+                "base_path": "/workspace/resnet-50-tf",
+                "layout": "NHWC:NCHW",
+                "shape": "(1,224,224,3)"
+           }
+        }
+    ],
+    "monitoring":
+        {
+            "metrics":
+            {
+                "enable" : true
+            }
+        }
+   }
+   ```
+
+## Restrict only certain metrics via config.json and CLI
+
+You can enable from one up to all the metrics available at once.
+
+To enable specific set of metrics you need to specify the metrics_list flag or json setting:
+
+CLI
+
+   ```bash
+         docker run --rm -d -v ${PWD}/models/resnet-50-tf:/opt/model -p 9001:9001 -p 9002:9002 openvino/model_server:latest \
+               --model_path /opt/model --model_name resnet --port 9001 \
+               --rest_port 9002 \
+               --metrics_enabled
+               --metrics_list ovms_requests_success,ovms_infer_req_queue_size
+   ```
+
+CONFIG CMD
+
+   ```bash
+         docker run --rm -d -v -d -v ${PWD}/workspace:/workspace openvino/model_server --config_path /workspace/config.json -p 9001:9001 -p 9002:9002 openvino/model_server:latest \
+               --rest_port 9002
+   ```
+
+CONFIG JSON
+
+   ```bash
+   {
+    "model_config_list": [
+        {
+           "config": {
+                "name": "resnet",
+                "base_path": "/workspace/resnet-50-tf",
+                "layout": "NHWC:NCHW",
+                "shape": "(1,224,224,3)"
+           }
+        }
+    ],
+    "monitoring":
+        {
+            "metrics":
+            {
+                "enable" : true,
+                "metrics_list": ["ovms_requests_success", "ovms_infer_req_queue_size"]
+            }
+        }
+    }
+    ```
+
+## Available metrics families
```

Comment:
let's drop such list in favor of a table with the list of metrics families

Reply:
Ok